tests/test_plugin.py

```diff
@@ -166,10 +165,33 @@ def test_current_window_is_main(browser, splinter_webdriver):
     assert browser.driver.current_window_is_main()
 
 
-def test_executable():
+def test_executable(testdir, request):
```

Comment:
We are not testing the real behaviour that we expect: given there are 2 tests, and some parametrization of the webdriver depends on the current test being run, then the browser  should be started 2 times with the expected 2 different parameterizations.